examples/basic/variables.tf

```diff
@@ -0,0 +1,281 @@
+variable "region" {
```

Comment:
@syphernl thanks for the PR!
Please move the example from `examples/basic` to `examples/complete` folder - we use it for all our examples that get tested on AWS by terratest in all modules